README.md

```diff
@@ -5,22 +5,23 @@
 This broker was inspired  by the [cf-containers-broker](https://github.com/cloudfoundry-community/cf-containers-broker). It supports Docker and Bosh-based service deployments. More details on the implemented Cloud Foundry contract can be found [here](http://docs.cloudfoundry.org/services/api.html). Read the [Big Picture](https://github.com/SAP/service-fabrik-broker/wiki/Big-Picture) behind Service Fabrik Broker.
 
 # Table of Contents
-1. [Local Development Setup](https://github.com/SAP/service-fabrik-broker#local-development-setup-ubuntu)
-2. [Installing Docker](https://github.com/SAP/service-fabrik-broker#installing-docker)
-3. [Installing NVM](https://github.com/SAP/service-fabrik-broker#installing-nvm)
-4. [Installing Bosh Lite](https://github.com/SAP/service-fabrik-broker#installing-bosh-lite)
-5. [Installing Cloud Foundry](https://github.com/SAP/service-fabrik-broker#installing-cloud-foundry)
-6. [Installing the Broker](https://github.com/SAP/service-fabrik-broker#installing-the-broker)
-7. [Launch the Broker](https://github.com/SAP/service-fabrik-broker#launch-the-broker)
-8. [Register the Broker](https://github.com/SAP/service-fabrik-broker#register-the-broker)
-9. [Upload Bosh Director Based Service Releases](https://github.com/SAP/service-fabrik-broker#upload-bosh-director-based-service-releases)
-10. [Run a Service Lifecycle](https://github.com/SAP/service-fabrik-broker#run-a-service-lifecycle)
-11. [How to obtain Support](https://github.com/SAP/service-fabrik-broker#how-to-obtain-support)
-12. [Advanced Debugging](https://github.com/SAP/service-fabrik-broker#advanced-debugging)
+1. [Local Development Setup](#local-development-setup-ubuntu)
+2. [Installing Docker](#installing-docker)
+3. [Installing NVM](#installing-nvm)
+4. [Installing Bosh Lite](#installing-bosh-lite)
+5. [Installing Cloud Foundry](#installing-cloud-foundry)
+6. [Installing the Broker](#installing-the-broker)
+7. [Launch the Broker](#launch-the-broker)
+8. [Register the Broker](#register-the-broker)
+9. [Launch the Managers](#launch-the-managers)
+10. [Upload Bosh Director Based Service Releases](#upload-bosh-director-based-service-releases)
```

Comment:
Provide one more step as local APIServer setup and running of it, that would be a prerequisite right?

Reply:
It is already mentioned below.
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/370/files/b39dc1c443d5ef7e373a053f297098b952f327ea#diff-04c6e90faac2675aa89e2176d2eec7d8L184

Reply:
ah!...missed it.